pkg/sanity/controller.go

```diff
@@ -269,16 +269,12 @@ var _ = Describe("DeleteVolume [Controller Server]", func() {
 		}
 	})
 
-	It("should fail when no volume id is provided", func() {
+	It("should not fail when no volume id is provided", func() {
```

Comment:
I don't think this is the correct behavior. See: https://github.com/container-storage-interface/spec/blob/master/spec.md#error-scheme

Reply:
I agree, I thought that it should fail.

Reply:
Gents, please check this part: https://github.com/container-storage-interface/spec/blob/master/spec.md#deletevolume
I guess there are to points here: 
1. When volume id is not provided
2. When volume id does not exist

So for the first one, we should return codes.InvalidArgument
but for the second one it should be OK

Would you agree?

Reply:
@sbezverk You are correct for DeleteVolume.

Reply:
I see, looks like the spec is double specifying behavior here. Looks like your code is good. Agree with @lpabon we should fix the spec soon.